tests/window/resizing.rs

```diff
@@ -22,8 +22,11 @@ fn main() {
         .add_plugins(
             DefaultPlugins.set(WindowPlugin {
                 primary_window: Some(Window {
-                    resolution: WindowResolution::new(MAX_WIDTH as f32, MAX_HEIGHT as f32)
-                        .with_scale_factor_override(1.0),
+                    resolution: WindowResolution::new(LogicalSize::new(
```

Comment:
This feels like a code smell: we should probably have an integer-based constructor.

Reply:
I agree. Having to write out that much code for this is a bit stinky. Maybe just reverting back to:
```rust
pub fn new(logical_width: u32, logical_height: u32) -> Self {

}
```
and changing the previous f32 to u32 kinda like how ickshonpe mentioned so we don't have to immediately convert?